internal/command/meta.go

```diff
@@ -34,8 +34,8 @@ type Meta struct {
 	Env *environment.CI
 	// shared go-tfe client
 	cloud *cloud.Cloud
-	// if github/gitlab context is not detected
-	cliOut map[string]string
+	// messages for stdout, platform output
+	messageOutput map[string]*OutputMessage
```

Comment:
Can we change the name to be plural since it is a map?

Reply:
Will change 👍